crypto/fipsmodule/ec/asm/p256-armv8-asm.pl

```diff
@@ -917,7 +917,7 @@
 	ldp	x23,x24,[x29,#48]
 	ldp	x25,x26,[x29,#64]
 	ldp	x27,x28,[x29,#80]
-	add	sp,sp,#32*(12-4)	// difference in stack frames
+	add	sp,sp,#256	// #256 is from #32*(12-4). difference in stack frames
```

Comment:
Did you end up needing this change, Bryce? Aren't there other similar changes that are you are now handled by the parser after you updated it?

Reply:
Yes, this is needed. Other syntaxes are addressed via adding new peg rules.

Reply:
But it looks like it should be handled by the rule in line 92 in delocate.peg, no?

Reply:
No, the line 92 requires the statement MUST contain `'['`.

```
ARMBaseIndexScale <- '[' ARMRegister (',' WS? (('#' Offset (('*' [0-9]+) / ('*' '(' [0-9]+ Operator [0-9]+ ')') / (('+' [0-9]+)*))? ) / ARMGOTLow12 / Low12BitsSymbolRef / ARMRegister) (',' WS? ARMConstantTweak)?)? ']' ARMPostincrement?
```